robosuite/environments/lift.py

```diff
@@ -319,9 +320,11 @@ def _reset_internal(self):
             # Sample from the placement initializer for all objects
             obj_pos, obj_quat = self.placement_initializer.sample()
 
+            # TODO: clean this up with respect to joint naming?
+
             # Loop through all objects and reset their positions
             for i, (obj_name, _) in enumerate(self.mujoco_objects.items()):
-                self.sim.data.set_joint_qpos(obj_name, np.concatenate([np.array(obj_pos[i]), np.array(obj_quat[i])]))
+                self.sim.data.set_joint_qpos(obj_name + "_0", np.concatenate([np.array(obj_pos[i]), np.array(obj_quat[i])]))
```

Comment:
where's the trailing "0" introduced? looks messy that we have to append this "0"  everywhere after the object name

Reply:
the reason is that every object gets a free joint by default (see [here](https://github.com/StanfordVL/robosuite/blob/v1_pi/robosuite/models/objects/objects.py#L285)) but more than one joint can be created for an object if desired. Each joint is numbered - take a look at [this](https://github.com/StanfordVL/robosuite/blob/v1_pi/robosuite/models/tasks/table_top_task.py#L89). I agree it's a little messy, but I think it's best to have straightforward naming for the object joints like this.

Reply:
I see. I wasn't aware that `obj_name + "_0"` is the name of a joint not an object. This is a little confusing. Maybe we can say `obj_name + "_jnt0"` instead for clarity?